### DISCUSSION TITLE
Still using slack?

### DISCUSSION THREAD

**Discussion by mdrie** [StaticJsCMS/static-cms#98] [Q&A] [answered]
I just followed the slack link on https://staticjscms.netlify.app/community and got "This link is no longer active"

The link is https://join.slack.com/t/static-cms/shared_invite/zt-1gvgnf5yv-E4sR17YnEcOy6fLFH9m7bQ

Are you still using the slack channel? (Wouldn't mind if not...)

If not, is there a replacement?

**Comment by KaneFreeman** [comment]
Thanks for pointing that out. There is a slack, but I think its me and one other person at the moment. I was thinking of switching over to Discord, and might take this opportunity to do so.

**Comment by KaneFreeman** [comment, marked as answer]
Here is the discord server link: https://discord.gg/ZWJM9pBMjj
